modules/ossm-smcp-prod.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-deploy-production.adoc
+
+[id="ossm-smcp-prod_{context}"]
+= Configuring your ServiceMeshControlPlane resource for production
+
+If you have installed a basic `ServiceMeshControlPlane` resource to test {ProductShortName}, you must configure it to production specifications before you deploy your service mesh in production.
```

Comment:
You know your users more than I, but can they deduce that configuring service mesh to production specification allows them to deploy service mesh in production? In short, the last bit seems somewhat superfluous to me. Would this rewording still make sense?

`...you must configure it to production specification.`